src/main/java/com/icthh/xm/tmf/ms/activation/config/KafkaOffsetsMetric.java

```diff
@@ -0,0 +1,147 @@
+package com.icthh.xm.tmf.ms.activation.config;
+
+import static com.icthh.xm.commons.config.client.repository.TenantListRepository.TENANTS_LIST_CONFIG_KEY;
+import static java.util.Arrays.asList;
+
+import com.codahale.metrics.Gauge;
+import com.codahale.metrics.Metric;
+import com.codahale.metrics.MetricSet;
+import com.icthh.xm.commons.config.client.repository.CommonConfigRepository;
+import com.icthh.xm.commons.config.client.repository.TenantListRepository;
+import com.icthh.xm.commons.config.domain.Configuration;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.kafka.clients.consumer.Consumer;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.OffsetAndMetadata;
+import org.apache.kafka.common.PartitionInfo;
+import org.apache.kafka.common.TopicPartition;
+import org.apache.kafka.common.serialization.ByteArrayDeserializer;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.cloud.stream.binder.kafka.properties.KafkaBinderConfigurationProperties;
+import org.springframework.kafka.core.ConsumerFactory;
+import org.springframework.kafka.core.DefaultKafkaConsumerFactory;
+import org.springframework.stereotype.Component;
+import org.springframework.util.ObjectUtils;
+
+@Slf4j
+@RequiredArgsConstructor
+@Component
+public class KafkaOffsetsMetric implements MetricSet {
```

Comment:
@tsardima seems i saw similar MetricSet in another project.
Should we move metrics to commons project? Or we need to try them separately first? 
@ishkurko @sergeysenja1992 what do you think?

Reply:
@tsardima maybe @amedvedchuk right, and we nedd to made configurable this class in commons

Reply:
like metrics class receive topic and group name

Reply:
may be not only kafka related, but also other metrics whose can be common for different microservices.

Reply:
too many dependecy of kafka in gradle, now better to don't move this class to commons
 in future if this mertics will be used in more places, then we will create common project for this metric